CHANGELOG.md

```diff
@@ -24,6 +24,7 @@
 - Log SOCKS5 configuration for IMAP like already done for SMTP #3964
 - Fix SOCKS5 usage for IMAP #3965
 - Exit from recently seen loop on interrupt channel errors to avoid busy looping #3966
+- Delete messages to the trash folder first if there's one #3972
```

Comment:
```suggestion
- Fix a problem with GMail where (auto-)deleted messages would get archived instead of deleted. Delete messages to the trash folder first if there's one #3972
```
not sure this exactly catpures it but i suggest to invest a bit more work into the changelog message after all the good work in this PR (thanks @link2xt and @iequidoo !). Somewhat detailed changelog messages are also helpful for communication with testers down the line.